myapp/app/assets/stylesheets/components/_pagination.scss

```diff
@@ -0,0 +1,34 @@
+.pagination {
+  margin: 20px auto 0;
+  width: 50%;
+  display: flex;
+  justify-content: center;
+
+  span {
+    text-align: center;
+    width: 50px;
+
+    :hover {
+      background-color: #f3f3f3;
+    }
+
+    a {
+      display: block;
+      width: 100%;
+      height: 100%;
+
+      :hover {
+        background-color: #f3f3f3;
+      }
+    }
+  }
+}
+
+.current {
+  color: #008CFF;
+}
+
+.gap {
+  background-color: white !important;
```

Comment:
I think it's better to avoid using `!important` as much as possible for  the maintainability.